.circleci/config.yml

```diff
@@ -379,6 +379,8 @@ memcached: &memcached
 
 jobs:
   "node-18":
+    environment:
+      COLLECTOR_ONLY: true  
```

Comment:
Ah, so that's how the grown ups set this flag. Makes me feel stupid for having set that via https://app.circleci.com/settings/project/github/instana/nodejs/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Finstana%2Fnodejs a while ago, which of course then affects all builds 🤦 

I still think this should be removed before merging ;-)